nodes/src/nodes/Sql.Query.BulkInsert.ts

```diff
@@ -0,0 +1,56 @@
+import { ModuleCompute, ModuleDefinition } from '@nodescript/core/types';
+
+import { BulkInsertBuilder, BulkInsertParams, BulkInsertResult } from '../lib/BulkInsertBuilder.js';
+
+export const module: ModuleDefinition<BulkInsertParams, BulkInsertResult> = {
+    version: '0.0.1',
+    moduleName: 'SQL / Query / Bulk Insert',
+    description: 'Converts an input array representing rows of data to a SQL INSERT query string and params array.',
+    keywords: ['sql', 'database', 'insert', 'bulk', 'query'],
+    params: {
+        vendor: {
```

Comment:
Here's a though: since this is vendor-aware, how about moving this into `SQL / Bulk Insert`, then accept a connection (from which you'd be able to infer the vendor) and execute the actual query rather than requiring two different nodes to do it? Wdyt?